helpers/cp_test.go

```diff
@@ -212,3 +212,25 @@ func (s *cpSuite) TestCopySpecialFileErrors(c *C) {
 	err := CopySpecialFile("no-such-file", "no-such-target")
 	c.Assert(err, ErrorMatches, "failed to copy device node:.*cp:.*stat.*no-such-file.*")
 }
+
+func (s *cpSuite) TestCopyPreserveAll(c *C) {
+	src := filepath.Join(c.MkDir(), "meep")
+	dst := filepath.Join(c.MkDir(), "copied-meep")
+
+	err := ioutil.WriteFile(src, []byte(nil), 0644)
+	c.Assert(err, IsNil)
+
+	// wait a wee bit to get different mtimes in the copy
```

Comment:
Tests that rely on timing are a bit tricky. Is there something that we could compare instead? If not then feel free to ignore this comment.